test/cli/video-cli.test.ts

```
@@ -0,0 +1,584 @@
+        `Cannot read video file: ${filePath}\n` +
+        `  Resolved path: ${absolutePath}\n` +
+        `  Please check file permissions.`;
+
+      expect(expectedError).toContain("Cannot read video file");
+      expect(expectedError).toContain("file permissions");
+    });
+  });
+
+  describe("Video Options Edge Cases", () => {
+    it("should handle quality at boundary values", () => {
+      expect(0 >= 0 && 0 <= 100).toBe(true); // Min boundary
```

Biome (2.4.10)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)